src/EnergyPlus/MixedAir.cc

```diff
@@ -3639,6 +3639,12 @@ namespace MixedAir {
                 MixedAirTempAtMinOAFlow = Node(this->RetNode).Temp;
             }
             this->MixedAirTempAtMinOAFlow = MixedAirTempAtMinOAFlow;
+
+            if (this->MixMassFlow > 0) {
+                // calc outdoor air minimum fraction based on actual mixed air flow
+                Real64 OutAirMinFracTemp = curAirLoopFlow.MinOutAir / this->MixMassFlow;
+                OutAirMinFrac = OutAirMinFracTemp;
+            }
```

Comment:
Rather than make this change here, why not change [line 3580](https://github.com/NREL/EnergyPlus/pull/6861/files#diff-19711df337cdc7fe75d3e87b4c0cd502R3580) to use `MixMassFlow` - or `curAirLoopFlow.SupFlow`. And then the calculation for `MechVentOutsideAirMinFrac` [here](https://github.com/NREL/EnergyPlus/pull/6861/files#diff-19711df337cdc7fe75d3e87b4c0cd502R3613) would also be changed to use `.SupFlow` or simply `SysSA`. Hopefully that doesn't break anything other than changing the values reported for "Air System Outdoor Air Minimum Flow Fraction" and fixing HX bypass problems.

Reply:
@mjwitte Thanks. I will try your suggestion and test it locally and see how it goes.

Reply:
@mjwitte Jus for your information. Since we are now passing a new OutAirMinFrac value calculated with different basis to CalcOAEconomizer function, then the new OutAirMinFrac value used [here](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/MixedAir.cc#L4563) may cause diffs.

Reply:
Yeah, this may need further work, because there's the issue of cycling fans and FixedMin which doesn't get applied until later (but seems it should be done before this).

Reply:
I think some of the checks should come after CalcOAEconomizer function call to cap the OA flow based on the various restrictions that may apply and the OA flow determined by the economizer.